src/components/Header/index.jsx

```diff
@@ -1,10 +1,156 @@
-import { Link } from "react-router-dom";
+import { useEffect, useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import axios from "axios";
+import { signInWithPopup, signOut } from "firebase/auth";
+import { auth, provider } from "../../config/firebase";
+import { useUserLoginStatusStore } from "../../store/store";
```

Comment:
약간의 개행으로 정리해주면 좋을 것 같습니다!
```suggestion
import { useEffect, useState } from "react";
import { Link, useNavigate } from "react-router-dom";

import axios from "axios";

import { signInWithPopup, signOut } from "firebase/auth";
import { auth, provider } from "../../config/firebase";
import { useUserLoginStatusStore } from "../../store/store";
```

Reply:
개행으로 구분하면 훨씬 보기 편하겠네요! 수정하겠습니다!